### PR TITLE
Add user check org org manager

### PR DESCRIPTION
**Pull request by rememberlenny**

### CODE REVIEW

File: static_src/components/users.jsx [hunk 1]
```diff
@@ -230,11 +234,16 @@ export default class Users extends React.Component {
   get userParentEntityUserSelector() {
     // only show something if in a space and if the user is a org manager
     // or space manager.
-    if (!this.isSpace ||
-      (!this.currentUserIsSpaceManager && !this.currentUserIsOrgManager)) {
+    if (!this.isSpace) {
       return null;
     }
 
+    if (!this.currentUserIsOrgManager) {
```
Comment (el-mapache) [inline static_src/components/users.jsx]: could this be `!this.currentUserIsOrgManager && !this.currentUserIsSpaceManager`?

File: static_src/components/users.jsx [hunk 2]
```diff
@@ -230,11 +234,16 @@ export default class Users extends React.Component {
   get userParentEntityUserSelector() {
     // only show something if in a space and if the user is a org manager
     // or space manager.
-    if (!this.isSpace ||
```
Comment (jcscottiii) [inline static_src/components/users.jsx]: could you revert this change and line 241-245
